7.1/Dockerfile

```diff
@@ -24,12 +24,12 @@ RUN docker-php-ext-install gd
 RUN docker-php-ext-install zip
 
 RUN apt-get install -y libicu-dev
-RUN docker-php-ext-install intl
```

Comment:
this line shouldn't be touched, and should be kept next to `RUN apt-get install -y libicu-dev`

same for the other files

Reply:
it's still there, just one line below. shouldn't effect anything

Reply:
you are right, but it affect the git history and shows a diff where no diff happend...